chromium_src/chrome/browser/ui/webui/settings/settings_import_data_handler.cc

```diff
@@ -1,22 +0,0 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "brave/browser/importer/brave_external_process_importer_host.h"
-#include "brave/common/pref_names.h"
-
-#define ExternalProcessImporterHost BraveExternalProcessImporterHost
-
-#define BRAVE_IMPORT_DATA \
```

Comment:
not sure I'm following what happened in the description, did you talk to whoever created this?

Reply:
cc: @simonhong https://github.com/brave/brave-core/commit/0dc9d8c84ac9170c5d0b172d9913411a66ab8b7e updated `chromium_src/chrome/browser/ui/webui/settings/import_data_handler.cc`, but also for some reason re-created `chromium_src/chrome/browser/ui/webui/settings/settings_import_data_handler.cc` which doesn't override anything, because `settings_import_data_handler.cc` was renamed to `import_data_handler.cc` (see the change here https://github.com/brave/brave-core/commit/ed3cc74c258418c57284530d09f1f426a7161436).

Reply:
Thanks for cleanup. This file was added again by my mistake.